dataactbroker/handlers/fileHandler.py

```diff
@@ -472,9 +472,13 @@ def checkD1File(self, submission_id=None):
             if result.status_code != 200:
                 raise ResponseException(result.data)
 
-        url = "" if d1_file.url is None else d1_file.url
-        error_message = "" if d1_file.error_message is None else d1_file.error_message
+        user_id = LoginSession.getName(session)
+        slash_index = d1_file.upload_file_name.rfind("/")
+        timestamped_filename = d1_file.upload_file_name[slash_index + 1:]
 
+        url = "" if status != "finished" else self.s3manager.getSignedUrl(path=str(user_id),
+                                                                          fileName=timestamped_filename, method="GET")
```

Comment:
For the path parameter, it works with and without so I think it's indifferent as to if it's provided or not. But we should be consistent either way.